cmd/bancheck/doc.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main contains the CLI used for detecting risky APIs.
+// See https://pkg.go.dev/github.com/google/go-safeweb/safehttp#hdr-Restricting_Risky_APIs
+// for a high level overview.
+//
+// Overview
+//
+// Bancheck is a program that implements a static analysis module and lets you run it
+// on go packages of your choice. Under the hood it uses the go/analysis package
+// https://pkg.go.dev/golang.org/x/tools/go/analysis which provides all the tools that
+// are needed for static code analysis. The tool resolves fully qualified function
+// and import names and checks them against a config file that defines banned tokens.
+//
+// Usage
+//
+// Apart from the standard https://pkg.go.dev/golang.org/x/tools/go/analysis#Analyzer flags
+// the command requires a config flag where a list of config files should be provided.
+//
+//  $ ./bancheck
+//  bannedAPI: Checks for usage of banned APIs
```

Comment:
bancheck?

Reply:
`go install` generates `bancheck` executable. Is there any other preferred option?